src/JustSaying/AwsTools/QueueCreation/RedrivePolicy.cs

```diff
@@ -23,11 +26,17 @@ protected RedrivePolicy()
     {
     }
 
-    // Cannot use System.Text.Json below as no public parameterless constructor. Change for v7?
-
     public override string ToString()
+#if NET8_0_OR_GREATER
+        => System.Text.Json.JsonSerializer.Serialize(this, JustSayingSerializationContext.Default.RedrivePolicy);
```

Comment:
`this` _might_ be problematic for AoT in case it's a derived type.

Reply:
Yeah, that's a good point, I think it'll end up serializing the base fields, but none added by the derived type.
At the moment, `RedrivePolicy` is only used internally, and I can't see a way that an API user could derive it and pass it through to any internal usage, but I could be missing it.

If that is the case, I don't think it would be a bad idea to `internal sealed` it.